cuda-flags.file

```diff
@@ -1,6 +1,21 @@
 ### FILE cuda-flags
 # define the CUDA compilation flags in a way that can be shared by SCRAM-based and regular tools
 
+# on X86 and Power, build support for Kepler, Pascal and Volta/Turing
+%ifarch x86_64 ppc64le
+%define cuda_arch 35 60 70
+%endif
+
+# on ARM, build Volta/Turing and the Xavier SoC
+%ifarch aarch64
+%define cuda_arch 70 72
+%endif
+
+
+# LLVM style for listing the supported CUDA compute architectures
+%define llvm_cuda_arch %(echo $(for ARCH in %cuda_arch; do echo "sm_$ARCH"; done) | sed -e"s/ /;/g")
```

Comment:
@fwyzard , here we should have `sed -e"s/ /,/g` . LLVM cmake configuration does this
```
  set(LIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES ${default_capabilities} CACHE STRING
    "List of CUDA Compute Capabilities to be used to compile the NVPTX device RTL.")
  string(REPLACE "," ";" nvptx_sm_list ${LIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES})

```

Reply:
and looks like we do not need to add `sm_` . usig `,` instead of `;` resulted in (note duplicate `sm_sm_` )

```
set(CUDA_NVCC_FLAGS  ;; -gencode;arch=compute_sm_35,code=sm_sm_35;-gencode;arch=compute_sm_60,code=sm_sm_60;-gencode;arch=compute_sm_70,code=sm_sm_70)
```